Translation/en/Text/regex.txt

```diff
@@ -10,9 +10,9 @@ sr:"^(.+)目 首の向き$"=$1 Eye orientation
 sr:"^会いに(.+)$"=$1 to meet
 sr:"^(.+)の運勢$"=$1 fortune
 sr:"^すごく(.+)！$"=Very $1!
-sr:"^告白して(.+)$"=Confess, $1.
-sr:"^告白(.+.)側$"=Confessed side
-sr:"^一緒に行動(.+)回数$"=Sex together
+sr:"^告白して(.+)$"=No confession yet
+sr:"^告白(.+.)側$"=Confession
+sr:"^一緒に行動(.+)回数$"=Interacted With
```

Comment:
What's the captured (`(.+)`) bit here?  If you're discarding it intentionally (no `$1`) change from `sr:` to `r:`.  Also what's up with `(.+.)`?

Reply:
Yeah these 3 look not right to me, why discard the capture?

Reply:
```suggestion
```
These shouldn't be regexes as it turns out, will be re-added later.